perma_web/requirements.txt

```diff
@@ -87,38 +85,38 @@ Pillow==3.3.1
 pip-tools==1.7
 py==1.4.31                # via pytest
 pyasn1==0.1.9             # via cryptography, requests
-pycparser==2.14           # via cffi
+pycparser==2.16           # via cffi
 pycrypto==2.6.1           # via paramiko
 pyflakes==1.0.0           # via flake8
-pyopenssl==16.0.0         # via certauth, ndg-httpsclient, requests
-pyScss==1.3.4             # via django-pipeline-compass
+pyopenssl==16.2.0         # via certauth, ndg-httpsclient, requests
+pyScss==1.3.4
 pytest-xdist==1.10
-pytest==2.9.1             # via pytest-xdist
+pytest==3.0.3             # via pytest-xdist
 python-dateutil==2.2
-python-mimeparse==1.5.2   # via django-tastypie
+python-mimeparse==1.6.0   # via django-tastypie
```

Comment:
The only changes related to node and webpack are the ones in `requirements.in` -- dropping `django-pipeline`, updating `whitenoise`, adding `django_webpack_loader`.

`requirements.txt` is created from `requirements.in` by running `pip-compile`, and I think what's happening here is it's freezing new versions of sub-dependencies for unrelated packages, since those packages specify their dependencies with a range.